docs/user/jms-selector-guide.md

```diff
@@ -0,0 +1,17 @@
+# JMS Selectors
+
+This broker is written based on AMQP broker semantics. In addition, to AMQP broker semantics 
+following JMS selector functionalities are supported as well.
+
+1. __x-filter-jms-selector__ header is used to define the selector expression.
+    When binding a queue to an exchange above additional header can be provided with a
+    valid JMS selector expression to enable message filtering.  
+2. Only the topic exchange supports JMS message selectors. 
```

Comment:
I thought we can use selectors with queues as well. What will happen if someone uses a selector with queues? Maybe we should document that too. WDYT?

Reply:
We can. But it won't behave as expected. I'll add a doc with details explaining this.